README.md

```diff
@@ -1,13 +1,122 @@
-# JPA Relationship Lab 3
+# JPA Relationship Lab 2
```

Comment:
Only here to serve as a reminder to maybe rename the repo on GitHub once merged so this doesn't become confusing later on to students and instructors.